2023-02-13 10:56:32,731 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 10:59:50,771 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:02:25,762 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:02:48,673 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:22:21,580 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:24:23,904 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:25:26,888 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:30:12,355 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:34:05,867 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:35:04,366 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:37:48,254 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:39:17,896 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-13 11:40:18,828 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 13:57:45,100 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 13:58:53,947 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:08:10,516 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:09:49,468 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:11:34,885 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:20:45,338 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:20:47,908 ERROR: Exception on /index [GET] [in D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py:1741]
Traceback (most recent call last):
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\webapp\routes.py", line 16, in index
    check = OthersCheck()
            ^^^^^^^^^^^
NameError: name 'OthersCheck' is not defined
2023-02-14 14:23:04,407 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:23:07,403 ERROR: Exception on /index [GET] [in D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py:1741]
Traceback (most recent call last):
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\webapp\routes.py", line 49, in index
    return render_template("index.html",
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\templating.py", line 147, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\templating.py", line 130, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
  File "D:\Coding\mircoblog\venv\Lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "D:\Coding\mircoblog\webapp\templates\index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "D:\Coding\mircoblog\webapp\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block 'doc'
    {%- block html %}
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block 'html'
    {% block body -%}
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block 'body'
    {% block content -%}
  File "D:\Coding\mircoblog\webapp\templates\base.html", line 65, in block 'content'
    {% block app_content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\webapp\templates\index.html", line 20, in block 'app_content'
    {{ wtf.quick_form(check) }}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\jinja2\environment.py", line 485, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'wtf' is undefined
2023-02-14 14:23:47,701 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:30:29,605 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:30:49,682 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:31:51,091 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:32:36,054 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:32:38,745 ERROR: Exception on /index [GET] [in D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py:1741]
Traceback (most recent call last):
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask\app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Coding\mircoblog\webapp\routes.py", line 32, in index
    check.data = True
    ^^^^^^^^^^
AttributeError: property 'data' of 'OthersCheckForm' object has no setter
2023-02-14 14:34:11,046 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-14 14:34:30,663 INFO: Microblog startup [in D:\Coding\mircoblog\webapp\__init__.py:63]
2023-02-15 20:33:27,706 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-15 20:35:30,302 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-15 20:46:17,511 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-15 20:52:29,009 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-15 21:03:59,831 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-15 21:04:31,537 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-16 20:29:12,342 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-17 03:02:12,999 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-17 03:02:18,181 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:63]
2023-02-17 15:45:58,700 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:64]
2023-02-17 16:24:00,252 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:64]
2023-02-17 16:58:18,582 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:64]
2023-02-17 16:58:26,783 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:64]
2023-02-17 16:58:35,406 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:64]
2023-02-17 16:59:51,692 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:64]
2023-02-17 16:59:58,367 INFO: Microblog startup [in D:\Coding\microblog\webapp\__init__.py:64]
